STEER/STEERBase/AliMagFastDip5k/AliMagFastDip5k.cxx

```diff
@@ -0,0 +1,11 @@
+#include "../AliMagFast.h"
```

Comment:
AliMagFast is common header and since in the aliroot installation we store all headers in the common $ALICE_ROOT/include/, this relative path may create problems.

Reply:
Perhaps #include <AliMagFast.h> seems better?

Reply:
I would say #include "AliMagFast.h"